smart-contracts/assembly/contracts/dns/dns.ts

```diff
@@ -293,3 +380,45 @@ export function isBlacklisted(binaryArgs: StaticArray<u8>): StaticArray<u8> {
 
   return new Args().add(isBlacklisted).serialize();
 }
+
+/**
+ * Delete a single entry from the DNS based on the given website name.
+ *
+ * @param binaryArgs - Website name in a binary format using Args.
+ */
+export function deleteEntryFromDNS(binaryArgs: StaticArray<u8>): void {
+  const websiteName = new Args(binaryArgs).nextString().unwrap();
+
+  // Ensure that the caller is the owner of the website
+  onlyWebsiteOwner(websiteName);
+
+  const websiteNameBytes = new Args().add(websiteName);
```

Comment:
dont you miss `.serialize()` here ? but tests seems to pass. so it seems okay... 
Instead of serialize `websiteName` why not just use `binaryArgs` which looks the same?